security/encrypt-cluster-pod-traffic.md

```diff
@@ -63,21 +63,28 @@ To install WireGuard on the default Amazon Machine Image (AMI):
    sudo yum install wireguard-dkms wireguard-tools -y
    ```
 
-Additionally, you will need to enable host-to-host encryption mode for WireGuard using the following command.
+Additionally, you may optionally enable host-to-host encryption mode for WireGuard using the following command.
 
 ```bash
-kubectl -n calico-system set env daemonset/calico-node --containers="calico-node" FELIX_WIREGUARDHOSTENCRYPTIONENABLED="true"
+calicoctl patch felixconfiguration default --type='merge' -p '{"spec": {"wireguardHostEncryptionEnabled": true}}'
 ```
+
+> **Warning**: `wireguardHostEncryptionEnabled` is an experimental flag that extends WireGuard encryption to host-network IP addresses. It is currently only supported on managed clusters such an EKS and AKS, where WireGuard *cannot* be enabled on the cluster's master node. Enabling this flag while WireGuard is enabled on the master node can lead to a broken cluster, and neworking deadlock.
```

Comment:
"such as" -> "deployed in" (we don't support non-EKS and non-AKS managed clusters, right?)

Reply:
also, master -> control plane here as well